### DISCUSSION TITLE
Frigate+ labeling Questions

### DISCUSSION THREAD

**Discussion by jsbowling42** [blakeblackshear/frigate#9294] [Ask A Question] [answered]
I have a couple Questions regarding this image:
![image](https://github.com/blakeblackshear/frigate/assets/35424582/671ad62b-fe13-42b8-8a0e-43aa5b665e67)

how many of these cars should get labeled as cars? 
how small is to small to label?
The car that has his headlights on where you cant see the shape of the car. would that get labeled a car?

As a general rule of thumb if i can tell its a car, i will label it. but some cars appear so small that the bounding boxes are hard to draw or wont get created when labeled in Frigate+ WebUI. I'm torn about the car with the headlights on and approaching the camera, as it fits the "I can recognize its a car" guideline I've been following. 

would all of these get labeled?
![image](https://github.com/blakeblackshear/frigate/assets/35424582/df191a76-c9fe-4548-9d68-202cd1d7a86f)

**Comment by NickM-27** [comment, marked as answer]
based on the second image that is probably what I would label as well. I try to avoid the headlights because that can look like so many different things and the shape of the actual car is not there
